### DISCUSSION TITLE
Fooocus_win64_2-1-60  can not work

### DISCUSSION THREAD

**Discussion by hotlml** [lllyasviel/Fooocus#784] [General]
Fooocus_win64_2-1-60  can not work 。
Could someone please inform me about the nature of the problem?

![004](https://github.com/lllyasviel/Fooocus/assets/74162310/74789102-2fa1-4c7b-b6e5-be66a5bf9068)

**Comment by barepixels** [comment]
It means Fooocus crashed.  Close the CMD window and relaunch Foocus

**Comment by hotlml** [comment]
It means Fooocus crashed. Close the CMD window and relaunch Foocus

Thank you for your answer. It cannot be used after multiple reboots.

**Reply by barepixels** [reply]
Am sorry, but maybe 4GB vram is too small for SDXL

**Comment by hotlml** [comment]
Am sorry, but maybe 4GB vram is too small for SDXL

may be
Memory Storage：16G
Nvidia GeForce GTX 960 ( 4 GB ）
